src/Razor/src/Microsoft.VisualStudio.Editor.Razor/Telemetry/VSTelemetryReporter.cs

```diff
@@ -60,6 +60,4 @@ public override void LogTrace(string? message, params object?[] args)
 
     public override void LogError(Exception exception, string? message, params object?[] args)
         => _logger?.LogError(exception, message, args);
-
-
```

Comment:
FYI: I think this will collide with my telemetry PR. I seem to remember cleaning that up.

Reply:
NOT IF I BEAT YOU TO IT!

(I'll undo)

Reply:
I won :smile: